tests/PortingAssistant.Client.IntegrationTests/RunPortingCorrectnessWithDotNetFramework.cs

```diff
@@ -0,0 +1,132 @@
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Diagnostics;
+using NUnit.Framework;
+using PortingAssistant.Client.IntegrationTests.TestUtils;
+
+namespace PortingAssistant.Client.IntegrationTests
+{
+    public class RunPortingCorrectnessWithDotNetFrameworkTests
+    {
+        private string testDirectoryRoot;
+        private string tmpTestFixturePath;
+        private string expectedPortedTestSolutionExtractionPath;
+        private string actualTestSolutionExtractionPath;
+
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            testDirectoryRoot = TestContext.CurrentContext.TestDirectory;
```

Comment:
This code is very similar to the section in RunAnalysisCorrectnessWithDotNetFramework.cs which creates the directory as well, is it possible to pull this out into its own function?

Reply:
> This code is very similar to the section in RunAnalysisCorrectnessWithDotNetFramework.cs which creates the directory as well, is it possible to pull this out into its own function?

Good point, it indeed has some code duplication here. We can add a testbase class with all the common variable and Setup method, and have the two test class inherit from it, wdyt?